services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_api/_volume_removal.py

```diff
@@ -0,0 +1,158 @@
+import logging
+from datetime import datetime
+
+from fastapi.encoders import jsonable_encoder
+from pydantic import parse_obj_as
+from servicelib.docker_utils import to_datetime
+from tenacity._asyncio import AsyncRetrying
+from tenacity.retry import retry_if_exception_type
+from tenacity.stop import stop_after_delay
+from tenacity.wait import wait_fixed
+
+from ....core.settings import DynamicSidecarSettings
+from ....models.schemas.constants import DYNAMIC_VOLUME_REMOVER_PREFIX
+from ..docker_service_specs.volume_remover import (
+    DockerVersion,
+    spec_volume_removal_service,
+)
+from ._utils import _RetryError, docker_client
+
+log = logging.getLogger(__name__)
+
+SERVICE_FINISHED_STATES: set[str] = {
+    "complete",
+    "failed",
+    "shutdown",
+    "rejected",
+    "orphaned",
+    "remove",
```

Comment:
are you sure it's "remove" and not "removed"?

Reply:
I'd say it's accurate https://docs.docker.com/engine/swarm/how-swarm-mode-works/swarm-task-states/